DuckDuckGo/MacBrowserWaitlistView.swift

```diff
@@ -0,0 +1,465 @@
+                    Text("duckduckgo.com/mac")
+                        .font(.custom("proximanova-bold", size: 17))
+                        .foregroundColor(.blue)
+                        .menuController(UserText.macWaitlistCopy) {
+                            action(.copyDownloadURLToPasteboard)
+                        }
+                        .padding(.top, 6)
+                    
+                    Text(UserText.macWaitlistInviteScreenStep2Title)
+                        .font(.custom("proximanova-bold", size: 17))
+                        .foregroundColor(.macWaitlistText)
+                        .padding(.top, 22)
+                        .padding(.bottom, 8)
+                    
+                    Text(UserText.macWaitlistInviteScreenStep2Description)
+                        .font(.custom("proximanova-regular", size: 17))
+                        .foregroundColor(.macWaitlistText)
+                        .lineSpacing(6)
+                    
+                    InviteCodeView(inviteCode: inviteCode)
+                        .menuController(UserText.macWaitlistCopy) {
+                            action(.copyInviteCodeToPasteboard)
+                        }
+                        .padding(.top, 28)
+                    
+                    Spacer(minLength: 24)
+                    
+                    Button(action: {
+                        action(.openShareSheet)
+                    }, label: {
+                        Image("Share")
+                            .foregroundColor(.macWaitlistText)
+                    })
+                        .padding(.bottom, 18)
+                        .sheet(isPresented: $showShareSheet, onDismiss: {
+                            // Nothing to do
+                        }, content: {
+                            ActivityViewController(activityItems: activityItems)
+                        })
+                }
+                .frame(maxWidth: .infinity, minHeight: proxy.size.height)
+                .padding([.leading, .trailing], 18)
+                .multilineTextAlignment(.center)
+            }
+        }
+    }
+    
+}
+
+private struct InviteCodeView: View {
+    
+    let inviteCode: String
+    
+    var body: some View {
+        VStack(spacing: 4) {
+            Text(UserText.macWaitlistInviteCode)
+                .font(.custom("proximanova-regular", size: 17))
+                .foregroundColor(.white)
+                .padding([.top, .bottom], 4)
+
+            Text(inviteCode)
+                .font(.system(size: 34, weight: .semibold, design: .monospaced))
+                .padding([.leading, .trailing], 18)
+                .padding([.top, .bottom], 6)
+                .foregroundColor(.black)
+                .background(Color.white)
+                .cornerRadius(4)
+        }
+        .padding(4)
+        .background(Color.macWaitlistGreen)
+        .cornerRadius(8)
+    }
+    
+}
+
+// MARK: - Generic Views
+ 
+struct HeaderView: View {
```

Comment:
Can this (and `RoundedButtonStyle`) be private? 🤔  - just thinking that's quite a generic name and might want to be re-used elsewhere